.github/workflows/checks.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Prepare environment
         run: |
-          ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata-all
+          ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata
```

Comment:
I plan to remove `netdata` as an option to this tool.

Reply:
Unless you're changing 'netdat-all' to be behave like the current 'netdata' option does, don't. We should not be unconditionally pulling in Node.js Python, and all of our Python dependencies on installs on user systems.